src/index.ts

```diff
@@ -54,6 +62,18 @@ bot
 
     if (bot.application) {
       const { id } = bot.application;
+
+      await api.put(Routes.applicationCommands(id), {
```

Comment:
Something else we may want to add, this is update-only. Might want to make sure we handle deleted commands too, by comparing `api.get(Routes.applicationCommands(id))` with the list of commands in the code